airtable.js

```diff
@@ -98,162 +96,126 @@ const toAirtableFormat = (record, table) => {
 // ******** CRUD ******** //
 // Given a table and a record object, create a record on Airtable.
 function createRecord(table, record) {
-  return new Promise(function(resolve, reject) {
-    const transformedRecord = toAirtableFormat(record, table);
-    base(table).create([{ fields: transformedRecord }], function(err, records) {
-      if (err) {
-        reject(err);
-        return;
-      }
-
-      const expectedLen = 1;
-      if (records.length !== expectedLen) {
-        reject(
-          new Error(
-            `${records.length} records returned from creating 1 record. Expected: ${expectedLen}`
-          )
-        );
-        return;
-      }
+  const transformedRecord = toAirtableFormat(record, table);
+  return new Promise(function process(resolve, reject) {
+    base(table)
+      .create([{ fields: transformedRecord }])
+      .then(records => {
+        const expectedLen = 1;
+        // TODO will this error ever be triggered? It seems like AirTable should catch and throw this error
```

Comment:
Not sure...would have to check the airtable API but I'd imagine you're right. Down to remove